src/lambda/local/detectLocalLambdas.ts

```diff
@@ -17,6 +17,8 @@ export interface LocalLambda {
     handler?: string
 }
 
+// TODO: remove?
+// Key difference: `detectLocalTemplates()` crawls workspace for template files...
```

Comment:
Just out of curiosity, were the outputs from this and detectLocalTemplates the same? I can't remember exactly how this functionality differed off the top of my head.

Reply:
They are the same. Also this was only used for nodejs , AFAIR. 

Of course this also did extra work: crawls the project for template.yaml files. But we don't need that anymore, right? Because of (1) the templateRegistry and (2) we are not auto-discovering template.yaml, instead it is specified in launch.json.